docs/en/routing/introduction.md

```diff
@@ -228,51 +268,56 @@ export default factory(function App() {
 
 ## Code splitting by route
 
-When using `@dojo/cli-build-app`, Dojo supports automatic code splitting by default for all top level outlets. This means that all widgets referenced within the `Outlet`s `renderer` will include a specific bundle for the outlet that will be loaded lazily when a user accesses the route.
+When using `@dojo/cli-build-app`, Dojo supports automatic code splitting by default for all top level routes. This means that all widgets referenced within the `Route`s `renderer` will include a specific bundle for the route that will be loaded lazily when a user accesses the route.
 
 To take advantage of the code splitting there are 4 rules:
 
 1.  The routing configuration needs to be the default export in the `src/routes.ts` module.
 2.  The widgets must be the default export of their module.
 3.  The `renderer` property must be defined inline.
-4.  The outlet `id` must be static and defined inline.
+4.  The `id` and `outlet` in the routing config and must be static and defined inline.
```

Comment:
Not sure if "routing config and" is missing the and's "what" or just an extra "and"